atlp-rwanda/rca-phantom-team2-bn
# feat(user): implement reset password functionality

#### What does this PR do?

- Implements reset password functionality

#### Description of Task to be completed?

- User can be able to reset their password

#### How should this be manually tested?

- Clone the repo

- checkout to branch `ft-reset-password`

- Run `npm install` to install modules

- Run `npm run dev` to start the development server

- In your API testing tool (postman recommended) call API `localhost:4000/api/users/forgot-password` to get the reset password email
- Check your email for the link and use it in your API testing tool

#### Any background context you want to provide?

- request body for forgot-password
  `{ email: "someone@example.com" } `
- request body for reset-password
 `{ newPassword: "Updated Password"}`

#### What are the relevant Pivotal Tracker/trello stories?

- link to my implemented [trello](https://trello.com/c/wTiXGhHJ/22-reset-password) story
